### PR TITLE
Add models classes

### PR DESCRIPTION
**Pull request by CrystallineButterfly**
Hello there, Thank you for the review! 🛸

This PR contains the completion of the tasks below: 

💎 Familiarize yourself with 'rails c' command ✅
💎 Open rails console for your project ✅
💎 Use other CRUD methods to find, update and delete entities ✅
💎 Create custom methods ✅
💎 Users ✅
💎 A method that returns the 3 most recent posts for a given user ✅
💎 Posts ✅
💎 A method that updates the posts counter for a user ✅
💎 A method that returns the 5 most recent comments for a given post ✅
💎 Comments ✅
💎 A method that updates the comments counter for a post ✅
💎 Likes ✅
💎 A method that updates the likes counter for a post ✅
💎 Go to rails c and check if your methods are working ✅

🚀 All the best- K 🚀

### CODE REVIEW

File: db/migrate/20220922091238_create_comments.rb [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
   def change
     create_table :comments do |t|
+      t.integer :author_id, foreign_key: true
+      t.integer :post_id, foreign_key: true
       t.text :text
 
       t.timestamps
     end
+    add_foreign_key :comments, :users, column: :author_id
+    add_index :comments, :author_id
```
Comment (GzimAsani) [inline db/migrate/20220922091238_create_comments.rb]: - [ ] I am getting an error when trying to run your migrations, kindly take a step back and check the migrations
image

File: app/models/comment.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class CreateComment < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+      t.timestamps
+    end
+    add_reference :comments, :users, null: false, foreign_key: true
+    add_reference :comments, :posts, null: false, foreign_key: true
+  end
+end
```
Comment (GzimAsani) [inline app/models/comment.rb]: - [x] Comment model needs to have  a method that updates the comments counter for a post.

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello @CrystallineButterfly            :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with your models  :+1:  
 
 ## Required Changes 

- [ ] Check the comment below :arrow_double_down: 






 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AnselemOdims — APPROVED** [review]
### Hi  @CrystallineButterfly 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by CrystallineButterfly** [comment]
Thank you bro! @AnselemOdims

**Review by AnselemOdims — APPROVED** [review]
### Hi  @CrystallineButterfly 🙋‍♀️, 
Good job implementing all the project requirements for this milestone/project.

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by CrystallineButterfly** [comment]
Thank you! @AnselemOdims

**Pull request merged**